components/wzQuicktext.js

```diff
@@ -149,7 +149,7 @@ wzQuicktext.prototype = {
     this.mOS = appInfo.OS;
 
     this.mPrefService = Components.classes["@mozilla.org/preferences-service;1"].getService(Components.interfaces.nsIPrefService);
-    this.mPrefBranch = this.mPrefService.getBranch("quicktext.");
+    this.mPrefBranch = this.mPrefService.getBranch("extensions.quicktext.");
```

Comment:
Does this mean old preferences are lost?

Reply:
Yes, I am not able to recover the old settings without violating the rule. Any suggestions?

Reply:
Ok, I see what I can do, will get back to you...

Reply:
Done. Removed all unused prefs (and related dead code). I did not migrate, collapseState and menuCollapse, because they are menu UI states, not settings.